printer_mods/JosAr/Klicky-Probe/README.md

```diff
@@ -0,0 +1,116 @@
+<p float="left">
+  <img src="Photos/probe_components.jpg" width="150" />
+  <img src="Photos/probe_install.jpg" width="150" />
+  <img src="Photos/Probe_topside.jpg" width="150" />
+  <img src="Photos/probe_v1_underside.jpg" width="150" />
+  <img src="Photos/probe_complete.jpg" width="150" />
+</p>
+
+The AB mount wires are also connected with pressure from the magnets, you can use the probe magnets as a template to insert the AB mount magnets, it is easier that way to don't insert the magnets the wrong way.
+<p float="left">
+  <img src="Photos/AB_Mount_wiring_1.jpg" width="150" />
+  <img src="Photos/AB_Mount_wiring_2.jpg" width="150" />
+  <img src="Photos/AB_Mount_wiring_3.jpg" width="150" />
+  <img src="Photos/AB_Mount_wiring_4.jpg" width="150" />
+  <img src="Photos/AB_Mount_wiring_complete.jpg" width="150" /> 
+</p>
+
+You will not lose Y travel on any configuration in the tests that were done.
+
+It is also recommended to glue the magnets in place, superglue is good.
+
+You will need to add macros to Klipper to be able to dock and undock the probe as necessary to do the Endstop (if necessary) and Quad Gantry Level, it is in the Klipper Macro directory.
+
+<img src="Photos/All_Klicky_probe_components.jpg" width="600" />
+
+Probe BOM:
+- 1x microswitch (the omron D2F-5 or D2F-5L (removing the lever) is recommended)
+- 2x M2x10 self tapping
+- 4x 6x3 magnets
+
+AB mount BOM:
+- 3x 6x3 magnets
+
+Probe Dock:
+- 1x 6x3 magnets
+- 2x M3x20
+
+Fixed Dock mount:
+- 2x M3 threaded insert M3x5x4
+- 2x M5x10
+- 2x M5 t-nut or equivalent
+
+or 
+
+variable Dock mount:
+- 10x M3 threaded insert M3x5x4
+- 8x M3x8
+- 2x M5x10
+- 2x M5 t-nut or equivalent
+
+If you would like to check a possibly more uptodate repository, check [here](https://github.com/jlas1/Klicky-Probe)
+
+The macro is based on a version provided by the user garrettwp on Discord, many thanks to him.
+I have tweaked it a lot.
+It is also originally  based on the great Annex magnet dockable probe macros "#Originally developed by Mental, modified for better use on K-series printers by RyanG and Trails" and can be found [here](https://github.com/Annex-Engineering/Annex-Engineering_Other_Printer_Mods/blob/master/All_Printers/Microswitch_Probe/Klipper_Macros/dockable_probe_macros.cfg)
+
+Would also like to thank the Voron discord community and VoronDesign for all the work that was and still is being made to maintain the Voron ecosystem.
+
+The probe accuracy output is something like this:
+probe accuracy results: maximum 6.430000, minimum 6.426250, range 0.003750, average 6.428750, median 6.428750, standard deviation 0.000791
+
+There is now an arrow on the probe telling you where should the switch pole be to have the correct offset.
+The probe offsets are:
+- z_offset = 6.42
+- x_offset: 0
+- y_offset: 19.75
+
+Assembled Klicky Probe
+<p>
+<img src="Photos/Voron_V2.4_300mm_back.jpg" />
+
+![Dock and undock video](Photos/Dock_and_Undock.mp4)
```

Comment:
This results in a broken image, it seems that github markdown doesn't support embedding videos.  Just remove the exclamation mark and make it a link.